exampleCourse/questions/element/multipleChoice/question.html

```diff
@@ -169,3 +169,29 @@
 
     </div>
 </div>
+
+<div class="card my-2">
+    <div class="card-header">
+        Part 7
+    </div>
+
+    <div class="card-body">
+
+        <pl-question-panel>
+            <p>Here, you can observe the behavior of the <code>hint</code> attribute in <code>pl-answer</code>.</p>
```

Comment:
You probably want to expand on what the hint is expected to convey, as a suggestion, and comment on what is expected to happen when an option with a hint is selected.